cdap-ui/app/cdap/cdap.html

```diff
@@ -16,9 +16,12 @@
 
 <!DOCTYPE html>
 <html lang="en">
-<head>
+<head profile="http://www.w3.org/2005/10/profile">
     <title></title>
 
+    <link rel="icon"
+      type="image/png"
+      href="/cdap_assets/img/favicon.png">
```

Comment:
this is forcing favicon to be from this path. We have defined a favicon path in theme. Can we make this dynamic?

Reply:
While configuring respective themes we need to make sure the favicon file name remains the same. Until now we had different names for favicon and trying to accommodate that in the code however a favicon is a very unique type of image being used by the browser. Doesn't make much sense to have different themes have different names for it. We can enforce this and have the environment setting up the theme to overwrite the existing file.

Reply:
then the favicon configuration should be removed from the theme file. The express server is also serving a favicon that should be removed

Reply:
Filed [CDAP-16099](https://issues.cask.co/browse/CDAP-16099) JIRA for cleanup to be done later.